source/collimation.f90

```diff
@@ -3257,27 +3187,23 @@ subroutine collimate_end_collimator(stracki)
 
 ! should name_sel(1:11) extended to allow longer names as done for
 ! coll the coll_ellipse.dat file !!!!!!!!
-  if(((cdb_cNameUC(icoll).eq.name_sel(1:mNameLen)).or.&
-      (cdb_cName(icoll).eq.name_sel(1:mNameLen))) .and. iturn.eq.1  ) then
+  if(chr_toLower(cdb_cName(icoll)) == name_sel .and. iturn == 1) then
```

Comment:
same comment as before about what should be lowered in order to perform the comparison

Reply:
It must be this way to work with both new and old inputs.